### PR TITLE
Fix email templating

### PR DESCRIPTION
**Pull request by Soldras**
* Change location email template
* Add init for all locale
* Add translation for email

### CODE REVIEW

File: I18n/email/default/en_US.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
```
Comment (Mertiozys) [inline I18n/email/default/en_US.php]: Translation key must be in english

File: I18n/email/default/fr_FR.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+
```
Comment (Mertiozys) [inline I18n/email/default/fr_FR.php]: Translation key must be in english

File: templates/email/default/payzen_payment_confirmation.html [hunk 3]
```diff
@@ -1,62 +1,65 @@
...
+            text-align: center;
+        }
+    </style>
+</head>
+
+<body>
+<div id="wrapper">
+    <div id="entete">
+        <h1 id="logotexte">{config key="store_name"}</h1>
+
```
Comment (Mertiozys) [inline templates/email/default/payzen_payment_confirmation.html]: Translation key must be in english

File: templates/email/default/payzen_payment_confirmation.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{default_translation_domain domain='payzen.email.default'}
+{default_locale locale={$locale}}
+{intl l="Cher client,"}
```
Comment (Mertiozys) [inline templates/email/default/payzen_payment_confirmation.txt]: Translation key must be in english

File: I18n/email/default/en_US.php (hunk 1 above)
Comment (roadster31) [inline I18n/email/default/en_US.php]: Why ?

Comment (lunika) [inline I18n/email/default/en_US.php]: Because all other keys are already in english ?

**Pull request merged**